src/OpenLoco/src/Windows/About.cpp

```diff
@@ -119,9 +115,15 @@ namespace OpenLoco::Ui::Windows::About
         drawingCtx.drawStringCentred(*rt, x, y, Colour::black, StringIds::licenced_to_atari_inc, nullptr);
     }
 
-    static void initEvents()
+    static constexpr WindowEventList _events = []() {
```

Comment:
why is it we have to do this with a lamda?

Reply:
Oh! We started out with that because we were using separate assignments per event (`events.onClick = onClick;` etc.). However, now that we're using designated initialisers, a lambda is likely no longer necessary.

Reply:
I think one of the reasons has been that some functions are declared below the usage so we typically have a declaration for getEvents at the top and implement it close to the bottom.